pkg/clients/cloudformation/cloudformation.go

```diff
@@ -33,7 +34,7 @@ type Client interface {
 }
 
 type cloudFormationClient struct {
-	cloudformation cfiface.CloudFormationAPI
+	cloudformation cfiface.ClientAPI
```

Comment:
What's the reason behind this interface change?

Reply:
The interfaces were renamed somewhere in the bump from v0.5 to v0.19 in the AWS SDK